linodecli/operation.py

```diff
@@ -346,6 +348,12 @@ def process_response_json(
         if self.response_model is None:
             return
 
+        override = OUTPUT_OVERRIDES.get(
+            (self.command, self.action, handler.mode)
+        )
+        if override is not None and not override(self, handler, json):
+            return
```

Comment:
Wonder why we need to check `not override(self, handler, json)` here. For example, if the override function return a `True` then we will have to run the rest of `process_response_json` function?

Reply:
> For example, if the override function return a True then we will have to run the rest of process_response_json function?

Yep! We're checking for the result to be `False` here so we can exit the function early if necessary